shared/fs/really-delete/index.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import {Box, ConfirmModal, HeaderOnMobile, Icon, MaybePopup, ProgressIndicator} from '../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
+import * as Types from '../../constants/types/fs'
+
+export type Props = {
+  _deleting: boolean,
+  onBack: () => void,
+  onDelete: () => void,
+  path: Types.Path,
+  title: string,
+}
+
+const _Spinner = (props: Props) => (
+  <MaybePopup onClose={props.onBack}>
+    <Box
+      style={{...globalStyles.flexBoxColumn, alignItems: 'center', flex: 1, padding: globalMargins.xlarge}}
+    >
+      <ProgressIndicator style={{width: globalMargins.medium}} />
+    </Box>
+  </MaybePopup>
+)
+const Spinner = HeaderOnMobile(_Spinner)
```

Comment:
I could't find where this is used. Did you plan to use this somewhere?

Reply:
I think I just copied this assuming it was boilerplate when I made this thing.. will remove, thanks for catching.